resources/com.yubico.yubioath.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.yubico.yubioath</id>
+  <launchable type="desktop-id">com.yubico.yubioath.desktop</launchable>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>BSD-2-Clause</project_license>
+
+  <name>Yubico Authenticator</name>
+  <summary>Graphical interface for displaying OATH codes with a YubiKey</summary>
+  <url type="homepage">https://developers.yubico.com/yubioath-desktop/</url>
+  <content_rating type="oars-1.1"/>
```

Comment:
It is part of Flatpak guideline https://github.com/flathub/flathub/wiki/AppData-Guidelines#oars-information.

It means that the app could be used for children or users who want to avoid NSFW content (like PEGI rating system).

Reply:
Thanks, it can stay in then.